src/SFA.DAS.LevyTransferMatching.Web/Models/Opportunities/IndexRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Opportunities
+{
+    public class IndexRequest : ApplyRequest
```

Comment:
I'm not clear on why `IndexRequest` would inherit from `ApplyRequest` - its properties don't seem to be relevant or used.

Reply:
Agree. I have made amend.